### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ngreenwald**
Added in details about copying over compiled c files for external mounting
Removed examples that didn't specify a TF version when starting docker
Added links to GCP instance with pre-configured dependencies, as well as links to docker and nvidia-docker

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,17 +66,18 @@ This will start a jupyter session, with several example notebooks detailing vari
 
 DeepCell uses `nvidia-docker` and `tensorflow` to enable GPU processing.
 
-### Build a local docker container
+If using GCP, there are pre-built images which come with CUDA, docker, and nvidia-docker pre-installed: 
+https://console.cloud.google.com/marketplace/details/nvidia-ngc-public/nvidia_gpu_cloud_image
```
Comment (willgraf) [inline README.md]: Can you please format the link into the text?  Maybe "there are pre-built images" would be a good place for a link, or the whole thing?  I don't want to see the whole URL.

File: README.md [hunk 2]
```diff
@@ -66,17 +66,18 @@ This will start a jupyter session, with several example notebooks detailing vari
...
 
-```bash
-git clone https://github.com/vanvalenlab/deepcell-tf.git
-cd deepcell-tf
-docker build -t $USER/deepcell-tf .
-```
+Otheriwise, these will need to be downloaded and installed:
+https://github.com/NVIDIA/nvidia-docker
+https://docs.docker.com/install/linux/docker-ce/debian/
```
Comment (willgraf) [inline README.md]: Can you please format the link into the text? I don't want to see the whole URL.

File: README.md [hunk 3]
```diff
@@ -95,13 +96,34 @@ NV_GPU='0' nvidia-docker run -it \
...
+NV_GPU='0' nvidia-docker run -it \
+  -p 8888:8888 \
+  $USER/deepcell-tf:latest
+  
+# Use ctrl-p, ctrl-q to exit the running docker image without shutting it down
+
+# Then, get the container_id after running the below command
+docker ps 
+
+# Replace the path/to/deepcell-tf below with the path to your local copy of deepcell-tf, and replace container_id with the container_id from the docker ps command above
```
Comment (willgraf) [inline README.md]: Can you make these into relative variables so users don't have to copy/paste the paths?  maybe you can name the docker container and reference it that way.

I would prefer this block of code (or any block of code) to be copy/pastable.

**Comment by ngreenwald** [comment]
Hey Will, I incorporated the changes you requested. Quick question: the example code near the top of the README uses nvidia-docker, do you think I should move the installation instructions for docker, nvidia-docker, etc up there or leave them where they are?

File: README.md (hunk 3 above)
Comment (ngreenwald) [reply README.md]: Good call!

File: README.md [hunk 4]
```diff
@@ -66,17 +66,18 @@ This will start a jupyter session, with several example notebooks detailing vari
...
+If using GCP, there are [pre-built images](https://console.cloud.google.com/marketplace/details/nvidia-ngc-public/nvidia_gpu_cloud_image) which come with CUDA, docker, and nvidia-docker pre-installed: 
 
-```bash
-git clone https://github.com/vanvalenlab/deepcell-tf.git
-cd deepcell-tf
-docker build -t $USER/deepcell-tf .
-```
 
-The tensorflow version can be overridden with the build-arg `TF_VERSION`.
+Otheriwise, you will need to install [docker](
```
Comment (willgraf) [inline README.md]: typo

File: README.md [hunk 5]
```diff
@@ -66,17 +66,18 @@ This will start a jupyter session, with several example notebooks detailing vari
 
 DeepCell uses `nvidia-docker` and `tensorflow` to enable GPU processing.
 
-### Build a local docker container
+If using GCP, there are [pre-built images](https://console.cloud.google.com/marketplace/details/nvidia-ngc-public/nvidia_gpu_cloud_image) which come with CUDA, docker, and nvidia-docker pre-installed:
```
Comment (willgraf) [inline README.md]: I don't think it should end with a colon. maybe a comma?

I'm also not sure if the spacing looks right, maybe it should be one big paragraph?  (Maybe not, they are different ideas).  Can you convince me one way or the other?

File: README.md [hunk 6]
```diff
@@ -66,17 +66,18 @@ This will start a jupyter session, with several example notebooks detailing vari
...
 
-```bash
-git clone https://github.com/vanvalenlab/deepcell-tf.git
-cd deepcell-tf
-docker build -t $USER/deepcell-tf .
-```
 
-The tensorflow version can be overridden with the build-arg `TF_VERSION`.
+Otheriwise, you will need to install [docker](
+https://docs.docker.com/install/linux/docker-ce/debian/), [nvidia-docker](https://github.com/NVIDIA/nvidia-docker), and [cuda](https://developer.nvidia.com/cuda-downloads) separately.
```
Comment (willgraf) [inline README.md]: CUDA should probably be capitalized.

File: README.md [hunk 7]
```diff
@@ -85,23 +86,40 @@ docker build --build-arg TF_VERSION=1.15.0-gpu -t $USER/deepcell-tf .
...
+  -p 8888:8888 \
+  $USER/deepcell-tf:latest
+  
+# Use ctrl-p, ctrl-q to exit the running docker image without shutting it down
+
+# Then, get the container_id corresponding to the running image of deepcell
+container_id=$(docker ps -q --filter ancestor="$USER/deepcell-tf") 
+
+# Copy the compiled c extensions into your local version of the codebase:
+docker cp "$CONTAINER_ID:/usr/local/lib/python3.6/dist-packages/deepcell/utils/compute_overlap.cpython-36m-x86_64-linux-gnu.so" deepcell/utils/compute_overlap.cpython-36m-x86_64-linux-gnu.so
```
Comment (willgraf) [inline README.md]: Does this actually run?  I would think `$CONTAINER_ID` and `container_id` do not match.

Comment (ngreenwald) [reply README.md]: doh, I switched between the two when testing it locally and writing up the readme

File: README.md (hunk 5 above)
Comment (ngreenwald) [reply README.md]: I think it's basically one idea, single paragraph is probably better

**Comment by ngreenwald** [comment]
Great, thanks! Just fixed

**Pull request merged**